deduce/integrationtests/test_structured_annotations.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+
+import deduce
+
+
+def get_annotations(raw_text: str) -> bool:
+    annotations = deduce.annotate_text_structured(raw_text)
+    for annotation in annotations:
+        if annotation.end_ix - annotation.start_ix != len(annotation.text_):
+            return False
+        original = raw_text[annotation.start_ix:annotation.end_ix].replace("<", "(").replace(">", ")")
+        if original != annotation.text_ and (annotation.tag != "INSTELLING" or original.lower() != annotation.text_):
+            return False
+    return True
+
+def test_get_annotations(texts_list: list) -> bool:
+    return all([get_annotations(text) for text in texts_list])
+
+if __name__ == "__main__":
+    filename = '/media/bigdata/2. Gebruik en toepassing van data/20_12_18_Pablo_Mosteiro_Anonymization/raw_texts/merged.csv'
```

Comment:
Just tagging you here to make sure you know this path is in here, since it will be here forever if it's merged to main

Reply:
Thanks for pointing it out. I had never meant to push this file at all; it was a mistake